spec/models/items_spec.rb

```diff
@@ -13,4 +14,13 @@
   it { should_not have_valid(:buy_price).when(nil) }
   it { should_not have_valid(:sell_price).when(nil) }
   it { should_not have_valid(:info_date).when(nil) }
+
+  describe "searching" do
+    it 'matches by id' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.